source/screens/caseScreens/CaseOverview.tsx

```diff
@@ -142,7 +142,9 @@ const computeCaseCardComponent = (caseData, navigation, authContext, extra) => {
   const statusType = caseData?.status?.type || "";
   const isNotStarted = statusType.includes("notStarted");
   const isOngoing = statusType.includes("ongoing");
-  const isCompletionRequired = statusType.includes("completionRequired");
+  const isCompletionRequired =
+    statusType.includes("completionRequired") ||
+    statusType.includes("active:completion:required");
```

Comment:
There is now two kind of completion status that a case can have

'Ansökan behöver kompletteras' = active:completionRequired:viva'
'Stickprovskontroll' = active:completionRandomCheckRequired:viva

I think you need to update the code to make the use of them in a correct manner. :)

Reply:
Can it still have the standard `completionRequired` status? The task only lists `active:completion:required` as the status to be added.

Reply:
We might need to align with each otherhere @Sven65

Reply:
@Sven65 Dont think that this PR is needed, you can close this one.